.github/workflows/site.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '8'
+          java-version: '11'
```

Comment:
We probably can use 17 too, no?

Reply:
We could, but that would be inconsistent with the Stapler and HPI Maven plugins.